projects/kit/components/like/like.style.less

```diff
@@ -0,0 +1,88 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+@keyframes tuiLikeScale {
```

Comment:
We should we move this file to /styles/components

Reply:
Okay, but why are we doing this?

Reply:
These are pure CSS components that rely on Angular only for quality of life such as attributes binding and icons resolving. But they can work without Angular and can be shared later with the react team.